snuba/migrations/0004_drop_first_release_id_from_group_attributes.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import UUID, Column
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.DropColumn(
+                storage_set=StorageSetKey.GROUP_ATTRIBUTES,
+                table_name="group_attributes_local",
+                column_name="group_first_release_id",
+                target=OperationTarget.LOCAL,
+            ),
+            operations.DropColumn(
+                storage_set=StorageSetKey.GROUP_ATTRIBUTES,
+                table_name="group_attributes_dist",
+                column_name="group_first_release_id",
+                target=OperationTarget.DISTRIBUTED,
+            ),
+        ]
+
+    def backwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.GROUP_ATTRIBUTES,
+                table_name="group_attributes_dist",
+                column=Column(
+                    "group_first_release_id",
+                    UUID(Modifiers(nullable=True)),
```

Comment:
Just checking here. Would UUID be the desired column type for the reverse operation (i.e. adding back the new column)? Or is UInt64 the correct type?

Reply:
It sounds like the type change didn't work? so I assumed it wasn't safe to rollback to that

Reply:
Ok, we can rollback to UUID. I suppose when we add back the column, we'll add it with the Uint64 type?

Reply:
yeah, that's the plan!